src/api/app/models/comment.rb

```diff
@@ -27,6 +21,23 @@ def to_s
   def create_notification(params = {})
     params[:commenter] = user.id
     params[:comment_body] = body
+    params[:commenters] = involved_users((commentable_type.underscore + '_id').to_sym, commentable.id)
```

Comment:
I do not like this. Can't you pass just the Object and derive the type from there inside `involved_users`?